lib/ruby_home/config/characteristics.yml

```diff
@@ -241,6 +253,23 @@
       '0': Not Charging
       '1': Charging
       '2': Not Chargeable
+- :name: :closed_captions
+  :description: Closed Captions
+  :uuid: 000000DD-0000-1000-8000-0026BB765291
+  :format: uint8
+  :unit: nil
+  :permissions:
+  - securedRead
+  :properties:
+  - read
+  - write
+  - cnotify
+  :constraints:
+    MaximumValue: 1
+    MinimumValue: 0
+    ValidValues:
+      '0': Disabled
+      '1': Enabled
```

Comment:
I changed the capitalization when I created mappings in the `constraints` section. This came from here: https://github.com/homebridge/HAP-NodeJS/blob/186ce56fab55b5868142e27b59af6c5ab53038d6/src/lib/gen/HomeKit-TV.ts#L80-L81